### PR TITLE
fix brooken name on tinkers table

### PR DESCRIPTION
**Pull request by bombcar**

### CODE REVIEW

File: src/main/resources/assets/jabba/lang/de_DE.lang [hunk 1]
```diff
@@ -74,5 +74,7 @@ text.jabba.waila.key.content=Fassinhalt
 text.jabba.waila.key.quantity=Item-Anzahl
 text.jabba.waila.key.stacks=Max Stacks
 
-text.jabba.tooltip.slots=Slots : 
+text.jabba.tooltip.slots=Slots :
 text.jabba.tooltip.empty=< Leer >
+
+mcp.mobius.betterbarrel=Barrel
```
Comment (glowredman) [inline src/main/resources/assets/jabba/lang/de_DE.lang]: Should be "Fass"

File: src/main/resources/assets/jabba/lang/ru_RU.lang [hunk 2]
```diff
@@ -79,6 +79,8 @@ text.jabba.waila.key.content=Содержимое бочки
...
 
-text.jabba.tooltip.slots.used=Использовано слотов : 
-text.jabba.tooltip.slots.provided=Слотов предоставлено : 
-text.jabba.tooltip.empty=< Пустая >
\ No newline at end of file
+text.jabba.tooltip.slots.used=Использовано слотов :
+text.jabba.tooltip.slots.provided=Слотов предоставлено :
+text.jabba.tooltip.empty=< Пустая >
+
+mcp.mobius.betterbarrel=Barrel
```
Comment (glowredman) [inline src/main/resources/assets/jabba/lang/ru_RU.lang:86]: You don't need to add this to each lang file, if it is missing, MC will use the en_US localization

File: dependencies.gradle [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
-    compile("com.github.GTNewHorizons:waila:1.5.18:dev")
-    compileOnly("curse.maven:minefactory-reloaded-66672:2366150")
-    compileOnly("com.github.GTNewHorizons:ForestryMC:4.4.4:dev")
-    compileOnly("com.mod-buildcraft:buildcraft:7.1.23:dev") // Needed by Forestry
+    compile('com.github.GTNewHorizons:waila:1.5.19:dev')
+
+    compileOnly('com.github.GTNewHorizons:ForestryMC:4.4.6:dev')
+    compileOnly('curse.maven:minefactory-reloaded-66672:2366150')
+
+    runtime('com.github.GTNewHorizons:TinkersConstruct:1.9.0.13-GTNH:dev')
```
Comment (glowredman) [inline dependencies.gradle]: Is it really needed to runtime TiCon just to test a single "fearure"?

File: src/main/resources/assets/jabba/lang/ru_RU.lang (hunk 2 above)
Comment (boubou19) [reply src/main/resources/assets/jabba/lang/ru_RU.lang:86]: yeah but if it's missing and someone wants to translate it, he has to do a diff with the en_US localization. Making it explicitely not translated is more convenient imo and doesn't change anything.

Comment (glowredman) [reply src/main/resources/assets/jabba/lang/ru_RU.lang:86]: Yeah okay that is true

File: dependencies.gradle (hunk 3 above)
Comment (bombcar) [reply dependencies.gradle]: Whoops meant to remove that after testing

Comment (glowredman) [reply dependencies.gradle]: In future just put the dev jar in your run/mods folder :P (along with all of its deps...)

Comment (bombcar) [reply dependencies.gradle]: Then I have to find the mod and download it etc etc - this + uodate_deps.py finds that all for me heh

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**